components/Layout/Footer.jsx

```diff
@@ -39,11 +39,9 @@ const LEARN_LINKS = [
   { title: 'Olas Stack', link: '/stack' },
 ];
 
-// const EXPLORE_LINKS = [
-//   { title: 'Use Cases', link: '/explore#use-cases' },
-//   { title: 'Chains', link: '/explore#chains' },
-//   { title: 'Dashboards', link: '/explore#dashboards' },
-// ];
+const EXPLORE_LINKS = [
+  { title: 'Use Cases', link: '/explore#use-cases' },
```

Comment:
If we call it "Use Cases" here, then in the submenu on the top it should have the same name

Reply:
Yes it's been added to the Resources submenu and footer now